api/src/main/java/tc/oc/pgm/api/StringUtils.java

```diff
@@ -1,4 +1,4 @@
-package tc.oc.pgm.util;
+package tc.oc.pgm.api;
 
```

Comment:
StringUtils shouldn't be in api, but its used in SettingsValue.

Reply:
https://github.com/PGMDev/PGM/blob/148a9d39ec9085468befe9c1844f4afa45b49862/api/src/main/java/tc/oc/pgm/api/setting/SettingValue.java#L97-L109 for reference